embeds.xml

```diff
@@ -0,0 +1,20 @@
+<Ui xmlns="http://www.blizzard.com/wow/ui/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blizzard.com/wow/ui/
+	..\FrameXML\UI.xsd">
+	<Script file="Libs\LibStub\LibStub.lua"/>
+    <Script file="Libs\LibDualSpec-1.0\LibDualSpec-1.0.lua"/>
```

Comment:
Can you fix the indentations so it's tabs and aligned? It might be aligned in your editor but the mix of tabs/spaces makes it look weird here, and tabs :D